src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -235,10 +244,16 @@ internal static WorkerInitResponse WorkerInitRequestHandler(WorkerInitRequest re
             response.Capabilities.Add("RpcHttpTriggerMetadataRemoved", bool.TrueString);
             response.Capabilities.Add("UseNullableValueDictionaryForHttp", bool.TrueString);
             response.Capabilities.Add("TypedDataCollection", bool.TrueString);
+            response.Capabilities.Add("HandlesWorkerTerminateMessage", bool.TrueString);
 
             return response;
         }
 
+        internal void WorkerTerminateRequestHandler(WorkerTerminate request)
+        {
+            _hostApplicationLifetime.StopApplication();
```

Comment:
Q: Do we write a log entry in host before/after making this GRPC call to record this action (something like "sending worker shutdown message" or so)